src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { Routes, Route } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { getMissions } from './Redux/missions/missionsSlice';
+import Layout from './components/Layout';
+import Rockets from './routes/Rockets';
+import Missions from './routes/Missions';
+import Profile from './routes/Profile';
+import './styles/style.css';
+import { rocketItems } from './Redux/RocketsSlice/RocketSlice';
+
+function App() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(rocketItems());
+  }, [dispatch]);
+
+  useEffect(() => {
+    dispatch(getMissions());
+  }, [dispatch]);
```

Comment:
- [ ] It will be great to only `getMissions` when you navigate to the missions page. At the moment with this implementation, you are getting both the missions and the rockets when the page first loads which is against the project requirement. 
NB: Ensure when you fix this, Make sure the missions are fetch only once ⬇️ 
![default view](https://user-images.githubusercontent.com/32361963/233598399-d6c70953-d3dd-4d1d-9c0b-80feb8448ad5.png)